runtime/runtime/src/ext.rs

```diff
@@ -178,11 +191,7 @@ impl<'a> External for RuntimeExt<'a> {
         let data_keys = self
             .trie_update
             .iter(&trie_key_parsers::get_raw_prefix_for_contract_data(self.account_id, prefix))
-            .map_err(|err| {
-                VMLogicError::InconsistentStateError(InconsistentStateError::StorageError(
-                    err.to_string(),
-                ))
-            })?
```

Comment:
@EgorKulikov this is the place where we unconditionally upgrade *any* StorageError to InconsistentStateError. Is this a bug?

Reply:
It does seem weird, I'll investigate

Reply:
Though enclosing method does not seem to be used anywhere in our code and External in not exposed from near-vm-logic, so maybe it is just some old dead code?

Reply:
Wait, no, it is exposed

Reply:
But it is still not used anywhere

Reply:
Thanks for looking into this! Indeed it seems that this is both buggy and dead code. I've added a comment to the function that it is dead. I'd probably say that we should just remove it completely (to make sure that, if we do need in the future, we add proper tests).